src/web/app/components/simple-modal/simple-modal.module.ts

```diff
@@ -1,6 +1,7 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
-import { ProgressBarModule } from '../../components/progress-bar/progress-bar.module';
+import { AjaxLoadingModule } from '../ajax-loading/ajax-loading.module';
+import { ProgressBarModule } from '../progress-bar/progress-bar.module';
```

Comment:
Was the progress-bar.module somehow repositioned?

Reply:
Yeap, sorted the ordering after adding `AjaxLoadingModule` because TSLint complains when the import is not sorted.

Reply:
Ah nvm. I see that the changed line actually got a redundant path. This is alright.